exercises/Exercise1.txt

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
You can review new code or changes I propose and put your comments in the way I actually do.